zipline/gens/tradesimulation.py

```diff
@@ -190,10 +201,34 @@ def on_exit():
             if algo.data_frequency == 'minute':
                 def execute_order_cancellation_policy():
                     algo.blotter.execute_cancel_policy(DAY_END)
+
+                def process_minute_capital_changes(dt):
```

Comment:
In this context, for legibility. That `once_a_day` refers to this function, and that `once_a_day` is  created earlier in `transform` and outside of the context, would imply (at least to me) that the deferred creation is necessary (but I am likewise unsure if it is.)

Wouldn't block this PR on it, but was mentioning since it is surprising.